microsoft/testsuites/mdatp/mdatp.py

```diff
@@ -0,0 +1,100 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+import pathlib
+
+from assertpy import fail
+
+from lisa import Node, TestCaseMetadata, TestSuite, TestSuiteMetadata
+from lisa.operating_system import Posix
+from lisa.sut_orchestrator import AZURE
+from lisa.testsuite import simple_requirement
+
+
+@TestSuiteMetadata(
+    area="mdatp",
+    category="functional",
+    description="""
+        Test to verify there are no pre-installed copies of mdatp.
+    """,
+)
+class MdatpSuite(TestSuite):
+    @TestCaseMetadata(
+        description="""
+           Check for mdatp endpoint/cloud install, dump config info.
+           Fails if mdatp is present in the image without explicitly
```

Comment:
What means "without explicitly installing it"? Does the logic detect it, and pass in this case?

Reply:
I guess it's an unneccesary adjective...
It means we are chekcing for the presence of mdatp without installing it. The test doesn't install it, so we can safely assume if it's present when the test runs, we didn't install it.

Reply:
I've changed a few things to make the failure path more clear, add comments about changes that might be needed later, and changed the description based on your feedback.